docs/Introduction/getting-started.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: nodes.liquid
+section: gettingStarted
+title: "Getting Started with Chainlink"
+date: Last Modified
+permalink: /getting-started/
+---
+
+<div>
+  <div class="markdown-body">
+    <div class="cl-section-header">
+      <h2>Start your journey with Chainlink</h2>
```

Comment:
On page this feels visually duplicative with the title "Getting Started with Chainlink". should we just pick one?

Reply:
Moved to the title.